paper/short/main_wc995.tex

```diff
@@ -594,14 +596,15 @@ \subsection{Stimuli}
 
 \subsection{Target Sounds}
 
-Two spectrally complex and complementary target sounds (``buzzing'' and ``roaring'') were downloaded from the American Tinnitus Association \cite{MeasuringTinnitusAmerican2022} and truncated to $500$ ms in duration (Fig. \ref{fig:spectra}).
+Two spectrally complex and complementary target sounds (``buzzing'' and ``roaring'') were downloaded from the American Tinnitus Association \cite{MeasuringTinnitusAmerican2022} and truncated to $500$ ms in duration (power-spectral densities are displayed in the botton subplots of Fig. \ref{fig:spectra}).
 
 \subsection{Experiment}
 
-Ten ($n=10$) normal-hearing subjects listened to $A-X$ trials containing a target sound ($A$) followed by a stimulus ($X$). $X$ was randomly generated for each trial, while $A$ remained the same within a block of $100$ trials (Fig. \ref{fig:experimentdiagram}).
+Ten ($n=10$) normal-hearing subjects listened to \textit{A-X} trials containing a target sound ($A$) followed by a stimulus ($X$). $X$ was randomly generated for each trial, while $A$ remained the same within a block of $100$ trials (Fig. \ref{fig:experimentdiagram}).
 Subjects completed two (2) blocks per target sound ($p=200$ total trials per subject).
 Subjects were told that some stimuli had $A$ embedded in them, and were instructed to respond ``yes'' to such stimuli, otherwise ``no.''
-Subjects listened over earphones at a self-determined comfortable level.
+Subjects listened over earphones at a self-determined comfortable level, which they adjusted themselves while listening to sample stimuli.
+Presentation level (in dB SPL) was not recorded.
```

Comment:
Define SPL? 
"(in dB sound pressure level (SPL))". Or just remove that bit: " Presentation level was not recorded". If it wasn't recorded, it doesn't matter what it wasn't recorded in.